schemas/src/elements/inputs/Input.ChoiceSet.json

```diff
@@ -13,6 +13,14 @@
       "description": "Allow multiple choices to be selected.",
       "default": false
     },
+    "placeholder": {
+      "type": "string",
+      "description": "Description of the input desired. Displayed when no choice has been selected.",
```

Comment:
nit: we could mention that we support place holder text only when isMultiSelect is set false.

Reply:
Ah great catch Joseph!! Updated :)